frontend/src/App.jsx

```diff
@@ -13,9 +13,9 @@ import { connectWallet, logout, checkForCRMToken } from 'services/wallet';
 import { saveTelegramUser, getTelegramUserWalletId } from 'services/telegram';
 import Documentation from 'pages/spotnet/documentation/Documentation';
 import Withdraw from 'pages/vault/withdraw/Withdraw';
-
+import useWalletStore from 'stores /useWalletStore';
```

Comment:
fix imports here and everywhere, seems like `useWalletStore` file doesn't exist?